packages/ButtonGroup/stories/examples/Sandbox.js

```diff
@@ -1,12 +1,13 @@
 /* eslint-disable react/button-has-type */
 
 import React from "react";
+import * as constants from "@paprika/constants/lib/Constants";
```

Comment:
This here is a problem. It's fine to create an internally shared package to centralize those constants, but if the consumers need to install this package into their apps to access constants, we're signing up for version mismatches in the future as the package gets updates.
I _think_ the alternative would be to re-export so that they'd access e.g. `ButtonGroup.kinds.PRIMARY`

Reply:
yeap that's the idea @alexzherdev , the constant package will hold only those constants that are "universal" and we are re-exporting all those via the component, so you could do something like:

```js
import Button from "@paprika/button"
function App() {
  const { kind, size } = Button.types
  return <Button kind={kinds.primary} size={size.medium} />
}

```

The constant package is just for internal/author use and we should write a disclaimer about that in the Readme file.